handlers/postgresql.go

```diff
@@ -0,0 +1,86 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published
+// by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+package handlers
+
+import (
+	"context"
+
+	"github.com/percona/pmm-managed/api"
+	"github.com/percona/pmm-managed/services/postgresql"
+	"github.com/percona/pmm-managed/utils/logger"
+)
+
+// PostgreSQLServer handles requests to manage postgreSQL nodes and services.
```

Comment:
Let's use the same spelling of PostgreSQL consistently.

```suggestion
// PostgreSQLServer handles requests to manage PostgreSQL instances.
```